src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user as User;
```

Comment:
Your app will broken if user will be null

```suggestion
  } = user;
```

Reply:
fixed this